src/dotnet-test-console/project.json

```diff
@@ -0,0 +1,38 @@
+﻿{
+  "version": "1.0.0-featmsbuild-*",
+  "buildOptions": {
+  	"keyFile": "../../tools/Key.snk",
+    "compile": {
+      "include": [
+        "**/*.cs",
+        "../dotnet/CommandLine/*.cs"
+      ]
+    }
+  },
+
+  "dependencies": {
+    "System.Diagnostics.TraceSource": "4.0.0",
+    "System.Diagnostics.TextWriterTraceListener": "4.0.0",
+    "Microsoft.DotNet.Cli.Utils": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.ProjectModel": {
+      "target": "project"
+    },
+    "Microsoft.Extensions.Testing.Abstractions": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.InternalAbstractions": {
+  	  "target": "project"
+  	},
+    "Microsoft.DotNet.PlatformAbstractions": "1.0.1-beta-000919"
+  },
+
+  "frameworks": {
+    "netstandard1.6": {
```

Comment:
Is the point of splitting this into its own assembly that it is a standalone executable?

Basically, is "dotnet-test-console" an executable or a library?  If it is an executable, I think it should target 'netcoreapp1.0', have "emitEntryPoint: true" and have a "Main" method.